lib/snaplist_view.dart

```diff
@@ -12,6 +12,7 @@ class SnapList extends StatefulWidget {
   final ScrollProgressUpdate progressUpdate;
   final PositionUpdate positionUpdate;
   final ScrollStart scrollStart;
+  final bool verticalScrolling;
```

Comment:
I already wrote the `Axis` comment.

Reply:
The Axis is used in list and part of the `dart:ui`, so it will be familiar enough and will not need additional dependencies. And I would love if you mention it in the readme =)